awx/api/generics.py

```diff
@@ -31,6 +31,9 @@
 from rest_framework.renderers import StaticHTMLRenderer, JSONRenderer
 from rest_framework.negotiation import DefaultContentNegotiation
 
+from jsonpointer import JsonPointerException
+from jsonpatch import JsonPatch, JsonPatchException
```

Comment:
is this missing a change to requirements/ somewhere

Reply:
Yep.